backend/qemu.pm

```diff
@@ -264,6 +279,20 @@ sub start_qemu() {
         symlink($i, "$basedir/l$i") or die "$!\n";
     }
 
+    if ($vars->{NICTYPE} eq "vde") {
+        if (system('vde_switch', '-d', '-s', "/tmp/openqa_vde$vars->{NIC_VLAN}.ctl") == 0) {
```

Comment:
it's only safe if the program creating the directory does some checks. otherwise an attacker may pre-create it as symlink. Just don't bother. Avoid /tmp, problem solved. We could use /var/lib/openqa/pool/net or something like that.